src/tests/ftest/dfuse/bash.py

```diff
@@ -138,6 +138,9 @@ def run_bashcmd(self, il_lib=None, compatible_mode=False):
                             f"cksum {fuse_root_dir}/src.c",
                             f"bzip2 -z {fuse_root_dir}/lib.a",
                             f"chmod u-r {fuse_root_dir}/lib.a.bz2",
+                            "fio --readwrite=randwrite --name=test --size=\"2M\" --directory "\
```

Comment:
it's probably better to add some functional FIO tests with more jobs (> 4) to exercise this better.